src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -979,9 +978,17 @@ internal static void ExcludeFilesForLibPackage(ICollection<IPackageFile> files)
             PathResolver.FilterPackageFiles(files, file => file.Path, _libPackageExcludes);
         }
 
-        internal static void ExcludeFilesForSymbolPackage(ICollection<IPackageFile> files)
+        internal static void ExcludeFilesForSymbolPackage(ICollection<IPackageFile> files, SymbolPackageFormat symbolPackageFormat)
         {
             PathResolver.FilterPackageFiles(files, file => file.Path, _symbolPackageExcludes);
+            if(symbolPackageFormat == SymbolPackageFormat.Snupkg)
+            {
+                var toRemove = files.Where(t => !string.Equals(Path.GetExtension(t.Path), ".pdb", StringComparison.OrdinalIgnoreCase)).ToList();
```

Comment:
>OrdinalIgnoreCase [](start = 115, length = 17)

Should this use `PathUtility.GetStringComparisonBasedOnOS`?

Reply:
same here. pdb extension in any casing would imply symbols.